README.md

```diff
@@ -64,29 +65,29 @@ resolved in the past.
 
   This layer depends on:
 
-  **openembedded-core**  
-  URI: git://git.openembedded.org/openembedded-core  
-  subdirectory: meta  
-  branch: master  
-  revision: HEAD  
-
-  **meta-openembedded (meta-oe)**  
-  URI: git://git.openembedded.org/meta-openembedded  
-  subdirectory: meta-oe  
-  branch: master  
-  revision: HEAD  
-
-  **meta-python** (since a6d7ba92645a112af358efd94ff3aa0c74985a51@meta-openembedded)  
-  URI: git://git.openembedded.org/meta-openembedded  
-  subdirectory: meta-python  
-  branch: master  
-  revision: HEAD  
-  
-  **meta-multimedia** (since fe44ac167a2a76531af3519f3889fce92024567b@meta-openembedded)  
-  URI: git://git.openembedded.org/meta-openembedded  
-  subdirectory: meta-multimedia  
-  branch: master  
-  revision: HEAD  
+  **openembedded-core**
+  URI: git://git.openembedded.org/openembedded-core
+  subdirectory: meta
+  branch: master
```

Comment:
I would update branch to thud only, because for warrior and master couple of the open PRs are needed to build most of the recipes here.

Reply:
I had deliberately left the body of the README as is (except for the removal of the extraneous whitespace done for me by my editor), but I'll change this.